pulp_container/app/redirects.py

```diff
@@ -112,7 +112,9 @@ def redirect_to_object_storage(self, artifact, return_media_type):
             "ResponseContentType": return_media_type,
             "ResponseContentDisposition": f"attachment;filename={filename}",
         }
-        content_url = artifact.file.storage.url(artifact.file.name, parameters=parameters)
+        content_url = artifact.file.storage.url(
+            artifact.file.name, parameters=parameters, http_method=self.request.method
+        )
```

Comment:
Should we add a check, that method is in `rest_framework.permissions.SAFE_METHODS`?

Reply:
there is no need to check for safe_methods, redirects are happening anyway only on head/get calls https://github.com/pulp/pulp_container/blob/main/pulp_container/app/registry_api.py#L800 and https://github.com/pulp/pulp_container/blob/main/pulp_container/app/registry_api.py#L832

Reply:
@m-bull we should add this to the core codepath as well https://github.com/pulp/pulpcore/blob/master/pulpcore/content/handler.py#L780